examples/test/qlib/SqlUtil/MysqlSqlUtil.qtest

```diff
@@ -178,6 +223,23 @@ class MysqlTest inherits SqlTestBase {
         return "";
     }
 
+    private testCreateTableWithAutoIncrementNegative() {
+        MysqlTestAutoIncrementNegativeSchema schema(table.getDatasource());
+        try {
+            schema.align(False, m_options.verbose);
+        }
+        catch (hash ex) {
+            assertEq("MYSQL-COLUMN-ERROR", ex.err, "exception check");
+            assertEq(True, ex.desc =~ /AUTO_INCREMENT can be used only with a subset of types/, "exception check");
+        }
```

Comment:
if the exception wouldn't be thrown, the test will also pass. use 
```
assertThrows("MYSQL-COLUMN-ERROR", 
              "AUTO_INCREMENT can be used only with a subset of types",
              sub() { schema.align(False, m_options.verbose); });
```

Reply:
done, sir!

Reply:
sir, there is the "resolve conversation" button, sir